package.json

```diff
@@ -10,7 +10,6 @@
     "babel-code-frame": "^6.26.0",
     "babel-core": "^6.0.0",
     "babel-eslint": "^10.0.0",
-    "babel-jest": "^23.0.1",
```

Comment:
I mentioned this in the Jest 24 PR, but for completeness: you don't need this dependency as you implement your own transformer which does not use `babel-jest`: https://github.com/facebook/react/blob/c4f0b93703be3d165de78db7a381d6cca2203b2f/scripts/jest/preprocessor.js

Reply:
removed in #16297